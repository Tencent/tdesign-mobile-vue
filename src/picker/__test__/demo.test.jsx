/**
 * 该文件为由脚本 `npm run test:demo` 自动生成，如需修改，执行脚本命令即可。请勿手写直接修改，否则会被覆盖
 */

import { mount } from '@vue/test-utils';
import areaVue from '@/picker/demos/area.vue';
import baseVue from '@/picker/demos/base.vue';
import cascadeVue from '@/picker/demos/cascade.vue';
import mobileVue from '@/picker/demos/mobile.vue';
import titleVue from '@/picker/demos/title.vue';

const mapper = {
  areaVue,
  baseVue,
  cascadeVue,
  mobileVue,
  titleVue,
};

describe('Picker', () => {
  Object.keys(mapper).forEach((demoName) => {
    it(`Picker ${demoName} demo works fine`, () => {
      const wrapper = mount(mapper[demoName]);
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
