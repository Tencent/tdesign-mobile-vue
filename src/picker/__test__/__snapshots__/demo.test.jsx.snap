// Vitest Snapshot v1

exports[`Picker > Picker areaVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="t-cell t-cell--middle"
  >
    <div
      class="t-cell__left"
    >
      <!--v-if-->
      <!---->
    </div>
    <div
      class="t-cell__title"
    >
      选择地区
      <!--v-if-->
      <!--v-if-->
    </div>
    <!--v-if-->
    <div
      class="t-cell__right"
    >
      <div
        class="t-cell__right-icon"
      >
        <svg
          class="t-icon t-icon-chevron-right"
          fill="none"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
        >
          <path
            d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
            fill="currentColor"
            fill-opacity="0.9"
          />
        </svg>
      </div>
    </div>
  </div>
  <teleport-stub
    disabled="false"
    to="[object HTMLBodyElement]"
  >
    <div
      class="t-overlay"
      style="z-index: 1000; transition-duration: 300ms; display: none;"
    >
      
      
    </div>
    <div
      class="t-popup t-popup--bottom"
      style="display: none;"
    >
      <!--v-if-->
      
      <div
        class="t-picker"
      >
        <div
          class="t-picker__toolbar"
        >
          <div
            class="t-picker__cancel"
          >
            取消
          </div>
          <div
            class="t-picker__title"
          />
          <div
            class="t-picker__confirm"
          >
            确认
          </div>
        </div>
        <!---->
        <div
          class="t-picker__main"
        >
          
          <div
            class="t-picker-item__group"
          >
            <ul
              class="t-picker-item"
              style="transform: translate3d(0, 80px, 0);"
            >
              
              <li
                class="t-picker-item__item t-picker-item__item--active"
              >
                北京市
              </li>
              <li
                class="t-picker-item__item"
              >
                广东省
              </li>
              
            </ul>
          </div>
          <div
            class="t-picker-item__group"
          >
            <ul
              class="t-picker-item"
              style="transform: translate3d(0, 80px, 0);"
            >
              
              
            </ul>
          </div>
          <div
            class="t-picker-item__group"
          >
            <ul
              class="t-picker-item"
              style="transform: translate3d(0, 80px, 0);"
            >
              
              
            </ul>
          </div>
          
          <div
            class="t-picker__mask t-picker__mask--top"
          />
          <div
            class="t-picker__mask t-picker__mask--bottom"
          />
          <div
            class="t-picker__indicator"
          />
        </div>
      </div>
      
    </div>
  </teleport-stub>
  
</div>
`;

exports[`Picker > Picker baseVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="t-cell t-cell--middle"
    data-v-049f57a8=""
  >
    <div
      class="t-cell__left"
    >
      <!--v-if-->
      <!---->
    </div>
    <div
      class="t-cell__title"
    >
      选择地区
      <!--v-if-->
      <!--v-if-->
    </div>
    <!--v-if-->
    <div
      class="t-cell__right"
    >
      <div
        class="t-cell__right-icon"
      >
        <svg
          class="t-icon t-icon-chevron-right"
          fill="none"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
        >
          <path
            d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
            fill="currentColor"
            fill-opacity="0.9"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="t-cell t-cell--middle"
    data-v-049f57a8=""
  >
    <div
      class="t-cell__left"
    >
      <!--v-if-->
      <!---->
    </div>
    <div
      class="t-cell__title"
    >
      选择时间
      <!--v-if-->
      <!--v-if-->
    </div>
    <!--v-if-->
    <div
      class="t-cell__right"
    >
      <div
        class="t-cell__right-icon"
      >
        <svg
          class="t-icon t-icon-chevron-right"
          fill="none"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
        >
          <path
            d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
            fill="currentColor"
            fill-opacity="0.9"
          />
        </svg>
      </div>
    </div>
  </div>
  <teleport-stub
    data-v-049f57a8=""
    disabled="false"
    to="[object HTMLBodyElement]"
  >
    <div
      class="t-overlay"
      style="z-index: 1000; transition-duration: 300ms; display: none;"
    >
      
      
    </div>
    <div
      class="t-popup t-popup--bottom"
      style="display: none;"
    >
      <!--v-if-->
      
      <div
        class="t-picker"
        data-v-049f57a8=""
      >
        <div
          class="t-picker__toolbar"
        >
          <div
            class="t-picker__cancel"
          >
            取消
          </div>
          <div
            class="t-picker__title"
          />
          <div
            class="t-picker__confirm"
          >
            确认
          </div>
        </div>
        <!---->
        <div
          class="t-picker__main"
        >
          
          <div
            class="t-picker-item__group"
          >
            <ul
              class="t-picker-item"
              style="transform: translate3d(0, 80px, 0);"
            >
              
              <li
                class="t-picker-item__item t-picker-item__item--active"
              >
                北京市
              </li>
              <li
                class="t-picker-item__item"
              >
                上海市
              </li>
              <li
                class="t-picker-item__item"
              >
                广州市
              </li>
              <li
                class="t-picker-item__item"
              >
                深圳市
              </li>
              <li
                class="t-picker-item__item"
              >
                杭州市
              </li>
              <li
                class="t-picker-item__item"
              >
                成都市
              </li>
              <li
                class="t-picker-item__item"
              >
                长沙市
              </li>
              
            </ul>
          </div>
          
          <div
            class="t-picker__mask t-picker__mask--top"
          />
          <div
            class="t-picker__mask t-picker__mask--bottom"
          />
          <div
            class="t-picker__indicator"
          />
        </div>
      </div>
      
    </div>
  </teleport-stub>
  <teleport-stub
    data-v-049f57a8=""
    disabled="false"
    to="[object HTMLBodyElement]"
  >
    <div
      class="t-overlay"
      style="z-index: 1000; transition-duration: 300ms; display: none;"
    >
      
      
    </div>
    <div
      class="t-popup t-popup--bottom"
      style="display: none;"
    >
      <!--v-if-->
      
      <div
        class="t-picker"
        data-v-049f57a8=""
      >
        <div
          class="t-picker__toolbar"
        >
          <div
            class="t-picker__cancel"
          >
            取消
          </div>
          <div
            class="t-picker__title"
          />
          <div
            class="t-picker__confirm"
          >
            确认
          </div>
        </div>
        <!---->
        <div
          class="t-picker__main"
        >
          
          <div
            class="t-picker-item__group"
          >
            <ul
              class="t-picker-item"
              style="transform: translate3d(0, 80px, 0);"
            >
              
              <li
                class="t-picker-item__item t-picker-item__item--active"
              >
                2023
              </li>
              <li
                class="t-picker-item__item"
              >
                2022
              </li>
              <li
                class="t-picker-item__item"
              >
                2021
              </li>
              <li
                class="t-picker-item__item"
              >
                2020
              </li>
              <li
                class="t-picker-item__item"
              >
                2019
              </li>
              <li
                class="t-picker-item__item"
              >
                2018
              </li>
              <li
                class="t-picker-item__item"
              >
                2017
              </li>
              <li
                class="t-picker-item__item"
              >
                2016
              </li>
              <li
                class="t-picker-item__item"
              >
                2015
              </li>
              <li
                class="t-picker-item__item"
              >
                2014
              </li>
              
            </ul>
          </div>
          <div
            class="t-picker-item__group"
          >
            <ul
              class="t-picker-item"
              style="transform: translate3d(0, 80px, 0);"
            >
              
              <li
                class="t-picker-item__item t-picker-item__item--active"
              >
                春
              </li>
              <li
                class="t-picker-item__item"
              >
                夏
              </li>
              <li
                class="t-picker-item__item"
              >
                秋
              </li>
              <li
                class="t-picker-item__item"
              >
                冬
              </li>
              
            </ul>
          </div>
          
          <div
            class="t-picker__mask t-picker__mask--top"
          />
          <div
            class="t-picker__mask t-picker__mask--bottom"
          />
          <div
            class="t-picker__indicator"
          />
        </div>
      </div>
      
    </div>
  </teleport-stub>
  
</div>
`;

exports[`Picker > Picker cascadeVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <!-- 联动选择 -->
  
  <!--v-if-->
  <div
    class="t-cell-group t-cell-group--default"
  >
    
    <div
      class="t-cell t-cell--middle"
    >
      <div
        class="t-cell__left"
      >
        <!--v-if-->
        <!---->
      </div>
      <div
        class="t-cell__title"
      >
        英雄
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell__note"
      >
        战士,狂铁
      </div>
      <div
        class="t-cell__right"
      >
        <div
          class="t-cell__right-icon"
        >
          <svg
            class="t-icon t-icon-chevron-right"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </div>
      </div>
    </div>
    
  </div>
  
  <teleport-stub
    disabled="false"
    to="[object HTMLBodyElement]"
  >
    <div
      class="t-overlay"
      style="z-index: 1000; transition-duration: 300ms; display: none;"
    >
      
      
    </div>
    <div
      class="t-popup t-popup--bottom"
      style="display: none;"
    >
      <!--v-if-->
      
      <div
        class="t-picker"
      >
        <div
          class="t-picker__toolbar"
        >
          <div
            class="t-picker__cancel"
          >
            取消
          </div>
          <div
            class="t-picker__title"
          >
            王者荣耀
          </div>
          <div
            class="t-picker__confirm"
          >
            确认
          </div>
        </div>
        <!---->
        <div
          class="t-picker__main"
        >
          
          <div
            class="t-picker-item__group"
          >
            <ul
              class="t-picker-item"
              style="transform: translate3d(0, 80px, 0);"
            >
              
              <li
                class="t-picker-item__item t-picker-item__item--active"
              >
                战士
              </li>
              <li
                class="t-picker-item__item"
              >
                法师
              </li>
              
            </ul>
          </div>
          <div
            class="t-picker-item__group"
          >
            <ul
              class="t-picker-item"
              style="transform: translate3d(0, -40px, 0);"
            >
              
              <li
                class="t-picker-item__item"
              >
                夏侯惇
              </li>
              <li
                class="t-picker-item__item"
              >
                吕布
              </li>
              <li
                class="t-picker-item__item"
              >
                铠
              </li>
              <li
                class="t-picker-item__item t-picker-item__item--active"
              >
                狂铁
              </li>
              
            </ul>
          </div>
          
          <div
            class="t-picker__mask t-picker__mask--top"
          />
          <div
            class="t-picker__mask t-picker__mask--bottom"
          />
          <div
            class="t-picker__indicator"
          />
        </div>
      </div>
      
    </div>
  </teleport-stub>
  
</div>
`;

exports[`Picker > Picker mobileVue demo works fine 1`] = `
<div
  class="tdesign-mobile-demo"
>
  <h1
    class="title"
  >
    Picker 选择器
  </h1>
  <p
    class="summary"
  >
    用于一组预设数据中的选择。
  </p>
  <div
    class="tdesign-mobile-demo-block"
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        01 组件类型
      </h2>
      <p
        class="tdesign-mobile-demo-block__summary"
      >
        基础选择器
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      
      <div
        class="t-cell t-cell--middle"
        data-v-049f57a8=""
      >
        <div
          class="t-cell__left"
        >
          <!--v-if-->
          <!---->
        </div>
        <div
          class="t-cell__title"
        >
          选择地区
          <!--v-if-->
          <!--v-if-->
        </div>
        <!--v-if-->
        <div
          class="t-cell__right"
        >
          <div
            class="t-cell__right-icon"
          >
            <svg
              class="t-icon t-icon-chevron-right"
              fill="none"
              height="1em"
              viewBox="0 0 16 16"
              width="1em"
            >
              <path
                d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
                fill="currentColor"
                fill-opacity="0.9"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        class="t-cell t-cell--middle"
        data-v-049f57a8=""
      >
        <div
          class="t-cell__left"
        >
          <!--v-if-->
          <!---->
        </div>
        <div
          class="t-cell__title"
        >
          选择时间
          <!--v-if-->
          <!--v-if-->
        </div>
        <!--v-if-->
        <div
          class="t-cell__right"
        >
          <div
            class="t-cell__right-icon"
          >
            <svg
              class="t-icon t-icon-chevron-right"
              fill="none"
              height="1em"
              viewBox="0 0 16 16"
              width="1em"
            >
              <path
                d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
                fill="currentColor"
                fill-opacity="0.9"
              />
            </svg>
          </div>
        </div>
      </div>
      <teleport-stub
        data-v-049f57a8=""
        disabled="false"
        to="[object HTMLBodyElement]"
      >
        <div
          class="t-overlay"
          style="z-index: 1000; transition-duration: 300ms; display: none;"
        >
          
          
        </div>
        <div
          class="t-popup t-popup--bottom"
          style="display: none;"
        >
          <!--v-if-->
          
          <div
            class="t-picker"
            data-v-049f57a8=""
          >
            <div
              class="t-picker__toolbar"
            >
              <div
                class="t-picker__cancel"
              >
                取消
              </div>
              <div
                class="t-picker__title"
              />
              <div
                class="t-picker__confirm"
              >
                确认
              </div>
            </div>
            <!---->
            <div
              class="t-picker__main"
            >
              
              <div
                class="t-picker-item__group"
              >
                <ul
                  class="t-picker-item"
                  style="transform: translate3d(0, 80px, 0);"
                >
                  
                  <li
                    class="t-picker-item__item t-picker-item__item--active"
                  >
                    北京市
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    上海市
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    广州市
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    深圳市
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    杭州市
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    成都市
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    长沙市
                  </li>
                  
                </ul>
              </div>
              
              <div
                class="t-picker__mask t-picker__mask--top"
              />
              <div
                class="t-picker__mask t-picker__mask--bottom"
              />
              <div
                class="t-picker__indicator"
              />
            </div>
          </div>
          
        </div>
      </teleport-stub>
      <teleport-stub
        data-v-049f57a8=""
        disabled="false"
        to="[object HTMLBodyElement]"
      >
        <div
          class="t-overlay"
          style="z-index: 1000; transition-duration: 300ms; display: none;"
        >
          
          
        </div>
        <div
          class="t-popup t-popup--bottom"
          style="display: none;"
        >
          <!--v-if-->
          
          <div
            class="t-picker"
            data-v-049f57a8=""
          >
            <div
              class="t-picker__toolbar"
            >
              <div
                class="t-picker__cancel"
              >
                取消
              </div>
              <div
                class="t-picker__title"
              />
              <div
                class="t-picker__confirm"
              >
                确认
              </div>
            </div>
            <!---->
            <div
              class="t-picker__main"
            >
              
              <div
                class="t-picker-item__group"
              >
                <ul
                  class="t-picker-item"
                  style="transform: translate3d(0, 80px, 0);"
                >
                  
                  <li
                    class="t-picker-item__item t-picker-item__item--active"
                  >
                    2023
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    2022
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    2021
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    2020
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    2019
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    2018
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    2017
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    2016
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    2015
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    2014
                  </li>
                  
                </ul>
              </div>
              <div
                class="t-picker-item__group"
              >
                <ul
                  class="t-picker-item"
                  style="transform: translate3d(0, 80px, 0);"
                >
                  
                  <li
                    class="t-picker-item__item t-picker-item__item--active"
                  >
                    春
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    夏
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    秋
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    冬
                  </li>
                  
                </ul>
              </div>
              
              <div
                class="t-picker__mask t-picker__mask--top"
              />
              <div
                class="t-picker__mask t-picker__mask--bottom"
              />
              <div
                class="t-picker__indicator"
              />
            </div>
          </div>
          
        </div>
      </teleport-stub>
      
      
      <div
        class="t-cell t-cell--middle"
      >
        <div
          class="t-cell__left"
        >
          <!--v-if-->
          <!---->
        </div>
        <div
          class="t-cell__title"
        >
          选择地区
          <!--v-if-->
          <!--v-if-->
        </div>
        <!--v-if-->
        <div
          class="t-cell__right"
        >
          <div
            class="t-cell__right-icon"
          >
            <svg
              class="t-icon t-icon-chevron-right"
              fill="none"
              height="1em"
              viewBox="0 0 16 16"
              width="1em"
            >
              <path
                d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
                fill="currentColor"
                fill-opacity="0.9"
              />
            </svg>
          </div>
        </div>
      </div>
      <teleport-stub
        disabled="false"
        to="[object HTMLBodyElement]"
      >
        <div
          class="t-overlay"
          style="z-index: 1000; transition-duration: 300ms; display: none;"
        >
          
          
        </div>
        <div
          class="t-popup t-popup--bottom"
          style="display: none;"
        >
          <!--v-if-->
          
          <div
            class="t-picker"
          >
            <div
              class="t-picker__toolbar"
            >
              <div
                class="t-picker__cancel"
              >
                取消
              </div>
              <div
                class="t-picker__title"
              />
              <div
                class="t-picker__confirm"
              >
                确认
              </div>
            </div>
            <!---->
            <div
              class="t-picker__main"
            >
              
              <div
                class="t-picker-item__group"
              >
                <ul
                  class="t-picker-item"
                  style="transform: translate3d(0, 80px, 0);"
                >
                  
                  <li
                    class="t-picker-item__item t-picker-item__item--active"
                  >
                    北京市
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    广东省
                  </li>
                  
                </ul>
              </div>
              <div
                class="t-picker-item__group"
              >
                <ul
                  class="t-picker-item"
                  style="transform: translate3d(0, 80px, 0);"
                >
                  
                  
                </ul>
              </div>
              <div
                class="t-picker-item__group"
              >
                <ul
                  class="t-picker-item"
                  style="transform: translate3d(0, 80px, 0);"
                >
                  
                  
                </ul>
              </div>
              
              <div
                class="t-picker__mask t-picker__mask--top"
              />
              <div
                class="t-picker__mask t-picker__mask--bottom"
              />
              <div
                class="t-picker__indicator"
              />
            </div>
          </div>
          
        </div>
      </teleport-stub>
      
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block"
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        02 组件样式
      </h2>
      <!--v-if-->
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      <!-- &lt;CasCadeDemo /&gt; -->
      
      <div
        class="t-cell t-cell--middle"
        data-v-7ee35211=""
      >
        <div
          class="t-cell__left"
        >
          <!--v-if-->
          <!---->
        </div>
        <div
          class="t-cell__title"
        >
          带标题选择器
          <!--v-if-->
          <!--v-if-->
        </div>
        <!--v-if-->
        <div
          class="t-cell__right"
        >
          <div
            class="t-cell__right-icon"
          >
            <svg
              class="t-icon t-icon-chevron-right"
              fill="none"
              height="1em"
              viewBox="0 0 16 16"
              width="1em"
            >
              <path
                d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
                fill="currentColor"
                fill-opacity="0.9"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        class="t-cell t-cell--middle"
        data-v-7ee35211=""
      >
        <div
          class="t-cell__left"
        >
          <!--v-if-->
          <!---->
        </div>
        <div
          class="t-cell__title"
        >
          无标题选择器
          <!--v-if-->
          <!--v-if-->
        </div>
        <!--v-if-->
        <div
          class="t-cell__right"
        >
          <div
            class="t-cell__right-icon"
          >
            <svg
              class="t-icon t-icon-chevron-right"
              fill="none"
              height="1em"
              viewBox="0 0 16 16"
              width="1em"
            >
              <path
                d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
                fill="currentColor"
                fill-opacity="0.9"
              />
            </svg>
          </div>
        </div>
      </div>
      <teleport-stub
        data-v-7ee35211=""
        disabled="false"
        to="[object HTMLBodyElement]"
      >
        <div
          class="t-overlay"
          style="z-index: 1000; transition-duration: 300ms; display: none;"
        >
          
          
        </div>
        <div
          class="t-popup t-popup--bottom"
          style="display: none;"
        >
          <!--v-if-->
          
          <div
            class="t-picker"
            data-v-7ee35211=""
          >
            <div
              class="t-picker__toolbar"
            >
              <div
                class="t-picker__cancel"
              >
                取消
              </div>
              <div
                class="t-picker__title"
              />
              <div
                class="t-picker__confirm"
              >
                确认
              </div>
            </div>
            <!---->
            <div
              class="t-picker__main"
            >
              
              <div
                class="t-picker-item__group"
              >
                <ul
                  class="t-picker-item"
                  style="transform: translate3d(0, 80px, 0);"
                >
                  
                  <li
                    class="t-picker-item__item t-picker-item__item--active"
                  >
                    北京市
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    上海市
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    广州市
                  </li>
                  <li
                    class="t-picker-item__item"
                  >
                    深圳市
                  </li>
                  
                </ul>
              </div>
              
              <div
                class="t-picker__mask t-picker__mask--top"
              />
              <div
                class="t-picker__mask t-picker__mask--bottom"
              />
              <div
                class="t-picker__indicator"
              />
            </div>
          </div>
          
        </div>
      </teleport-stub>
      
      
    </div>
  </div>
</div>
`;

exports[`Picker > Picker titleVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="t-cell t-cell--middle"
    data-v-7ee35211=""
  >
    <div
      class="t-cell__left"
    >
      <!--v-if-->
      <!---->
    </div>
    <div
      class="t-cell__title"
    >
      带标题选择器
      <!--v-if-->
      <!--v-if-->
    </div>
    <!--v-if-->
    <div
      class="t-cell__right"
    >
      <div
        class="t-cell__right-icon"
      >
        <svg
          class="t-icon t-icon-chevron-right"
          fill="none"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
        >
          <path
            d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
            fill="currentColor"
            fill-opacity="0.9"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="t-cell t-cell--middle"
    data-v-7ee35211=""
  >
    <div
      class="t-cell__left"
    >
      <!--v-if-->
      <!---->
    </div>
    <div
      class="t-cell__title"
    >
      无标题选择器
      <!--v-if-->
      <!--v-if-->
    </div>
    <!--v-if-->
    <div
      class="t-cell__right"
    >
      <div
        class="t-cell__right-icon"
      >
        <svg
          class="t-icon t-icon-chevron-right"
          fill="none"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
        >
          <path
            d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
            fill="currentColor"
            fill-opacity="0.9"
          />
        </svg>
      </div>
    </div>
  </div>
  <teleport-stub
    data-v-7ee35211=""
    disabled="false"
    to="[object HTMLBodyElement]"
  >
    <div
      class="t-overlay"
      style="z-index: 1000; transition-duration: 300ms; display: none;"
    >
      
      
    </div>
    <div
      class="t-popup t-popup--bottom"
      style="display: none;"
    >
      <!--v-if-->
      
      <div
        class="t-picker"
        data-v-7ee35211=""
      >
        <div
          class="t-picker__toolbar"
        >
          <div
            class="t-picker__cancel"
          >
            取消
          </div>
          <div
            class="t-picker__title"
          />
          <div
            class="t-picker__confirm"
          >
            确认
          </div>
        </div>
        <!---->
        <div
          class="t-picker__main"
        >
          
          <div
            class="t-picker-item__group"
          >
            <ul
              class="t-picker-item"
              style="transform: translate3d(0, 80px, 0);"
            >
              
              <li
                class="t-picker-item__item t-picker-item__item--active"
              >
                北京市
              </li>
              <li
                class="t-picker-item__item"
              >
                上海市
              </li>
              <li
                class="t-picker-item__item"
              >
                广州市
              </li>
              <li
                class="t-picker-item__item"
              >
                深圳市
              </li>
              
            </ul>
          </div>
          
          <div
            class="t-picker__mask t-picker__mask--top"
          />
          <div
            class="t-picker__mask t-picker__mask--bottom"
          />
          <div
            class="t-picker__indicator"
          />
        </div>
      </div>
      
    </div>
  </teleport-stub>
  
</div>
`;
